Lib/http/server.py

```diff
@@ -699,6 +701,7 @@ def send_head(self):
         """
         path = self.translate_path(self.path)
         f = None
+        self.range = self.parse_range()
```

Comment:
Since this attribute is "temporary" and may change at every request, I'm wondering whether a private writable attribute would be better (e.g., `self._range`) but could be publicly exposed via a read-only property, say `range`m (or any other name).